### DISCUSSION TITLE
Continuous Integration for bunjs

### DISCUSSION THREAD

**Discussion by haseeb-ul-hassan-123** [community/community#136323] [Actions]
### Select Topic Area

Question

### Body

can someone guide me step by step guide to create a ci/cd for bunjs ?

**Comment by hi-ammad** [comment]
name: bun-types
jobs:
  build:
    name: build-app
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repo
        uses: actions/checkout@v4
      - name: Install bun
        uses: oven-sh/setup-bun@v1
      - name: Install dependencies
        run: bun install
      - name: Build app
        run: bun run build
